frameworks/Java/servicetalk/src/main/java/servicetalk/Server.java

```diff
@@ -0,0 +1,100 @@
+         */
+        AsyncContext.disable();
+
+        /**
+        *   Factory to implement io pooling
+        */
+        IoExecutor ioExecutor = NettyIoExecutors.createIoExecutor(
+          Runtime.getRuntime().availableProcessors(),
+          new IoThreadFactory("io-pool")
+        );
+
+        /**
+         * Factory to disable headers validation
+         */
+
+        DefaultHttpHeadersFactory headersFactory = new DefaultHttpHeadersFactory(false,false);
+
+        try{
+            HttpSerializationProvider serializer = HttpSerializationProviders.jsonSerializer(new JacksonSerializationProvider());
+//            Create a custom server builder with performance enhancements
+            HttpServers
+              .forPort(8080)
+              .executionStrategy(HttpExecutionStrategies.noOffloadsStrategy())
+              .ioExecutor(ioExecutor)
+              .disableDrainingRequestPayloadBody()
+              .protocols(HttpProtocolConfigs.h1().headersFactory(headersFactory).build())
+              .appendConnectionAcceptorFilter(delegate -> new ConnectionAcceptor() {
+                  @Override
+                  public Completable accept(ConnectionContext context) {
+                      ((NettyConnectionContext)context).updateFlushStrategy((current, isOrig) -> FlushStrategies.flushOnEnd());
+                      return delegate.accept(context);
+                  }
+              })
+              .listenAndAwait((ctx, request, responseFactory) -> {
+                  ((NettyConnectionContext)ctx).updateFlushStrategy(((current, isCurrentOriginal) -> FlushStrategies.flushOnEach()));
+                  if(request.path().equals("/json")) {
+                      Map<String, String> obj = new HashMap<String, String>();
+                      obj.put("message", "Hello, World!");
+                      return succeeded(responseFactory.ok()
+                        .payloadBody(obj, serializer.serializerFor(Map.class))
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                  }
+
+                  if(request.path().equals("/plaintext")) {
+                      return succeeded(responseFactory.ok()
+                        .payloadBody("Hello, World!", HttpSerializationProviders.textSerializer())
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                    };
+                    return null;
+                })
+                .awaitShutdown();
+        } catch (Exception e) {
+//            logging is not allowed
+        }
+    }
+
+    public static String GetCurrentTime() {
+        SimpleDateFormat formatter = new SimpleDateFormat("EE, dd MMM yyyy kk:mm:ss z  ");
+        Date date = new Date();
+        return formatter.format(date);
+    }
+
+    private static int GetRandomNumber() {
+        Random random = new Random();
+        return random.nextInt(10001) + 1;
```

Comment:
This can return 10001, which is an invalid world id.  Also, you should use `ThreadLocalRandom`, which shares one `Random` instance with each thread instead of constructing a new `Random` instance for every number.  Do something like this instead:  https://github.com/TechEmpower/FrameworkBenchmarks/blob/f510e947b5480b83cdda66f397d31be7166a40d6/frameworks/Java/undertow/src/main/java/hello/HelloWebServer.java#L255